### DISCUSSION TITLE
How to embed a google slide to editor?

### DISCUSSION THREAD

**Discussion by MNhat0308** [awcodes/filament-tiptap-editor#302] [Q&A] [answered]
hi everyone,

to begin with I want to thanks your to make this package.

I can' t not embed a google slide to editor?

**Comment by awcodes** [comment, marked as answer]
I don't know what the markup is for a Google slide. But Prosemirror is very strict about what it allowed in the editor and what isn't.

More than likely you would need to make a custom Tiptap extension to be able to achieve this.

**Reply by MNhat0308** [reply]
big thank to you

**Comment by MNhat0308** [comment]
Hi sir,
I have make a custom extension that view the google slide using embed iframe like extension youtube in source code, but now i don't know how to trigger modal to get a  link google share slide that user type in
 i am hardcode the link in code

**Reply by awcodes** [reply]
Check the readme on custom extensions and blocks.

**Reply by MNhat0308** [reply]
thanks so much , i done it

**Reply by silverhand7** [reply]
@MNhat0308 I'm trying to make the same thing; is it possible for you to share the whole code for this?
